__tests__/app.test.js

```diff
@@ -80,4 +80,42 @@ describe("GET /api/categories", () => {
         });
     });
   });
+
+  describe("GET /api/users", () => {
+    test("should respond with an array of objects ", () => {
+      return request(app)
+        .get("/api/users")
+        .then(({ body: users }) => {
+          expect(Array.isArray(users)).toBe(true);
```

Comment:
when working with arrays it's nice to check the length of the array is what we expect it should be